src/Umbraco.Web.UI.Client/src/packages/tiny-mce/property-editors/stylesheets/property-editor-ui-tiny-mce-stylesheets-configuration.element.ts

```diff
@@ -38,6 +39,16 @@ export class UmbPropertyEditorUITinyMceStylesheetsConfigurationElement
 		this.dispatchEvent(new UmbPropertyValueChangeEvent());
 	}
 
+	constructor() {
+		super();
+		new UmbDeprecation({
+			deprecated: 'umb-property-editor-ui-tiny-mce-stylesheets-configuration',
+			removeInVersion: '16.0.0',
+			solution:
+				"Use `<umb-property-editor-ui-stylesheet-picker>` instead, or the 'Umb.PropertyEditorUi.StylesheetPicker' manifest.",
```

Comment:
I wonder if it's possible to get component name without hardcoded string?

Perhaps a `const` for component name?
https://stackoverflow.com/a/72343376

Reply:
Okay, perhaps sometimes later, but yeah I think some automatic docs tools don't like it 🙈

Reply:
I know that you can use the `@element my-element-name` jsdocs in some cases. Guess we need an overview of what would affect what if we changed anything.